opentreemap/treemap/lib/map_feature.py

```diff
@@ -315,8 +315,8 @@ def context_dict_for_resource(request, resource, **kwargs):
         __, display_area = get_display_value(instance,
                                              'greenInfrastructure', 'area',
                                              resource.calculate_area())
-        display_units = get_unit_name(get_units(instance,
-                                      'greenInfrastructure', 'area'))
+        display_units = get_unit_abbreviation(
```

Comment:
There's not a display problem, this is just changing to displaying unit abbreviations (which we do everywhere else).